sdks/python/apache_beam/io/external/xlang_jdbcio_it_test.py

```diff
@@ -191,6 +191,7 @@ def test_xlang_jdbc_write_read(self, database):
               username=self.username,
               password=self.password,
               classpath=classpath,
+              write_batch_size=5,
```

Comment:
can we remove this additional config? This is to test the IO works with minimum configuration provided, and sometimes user refers to the existing test to write their pipeline